### PR TITLE
Docker size reduction

### PR DESCRIPTION
**Pull request by AllMightLegend**

### CODE REVIEW

**Comment by Jaifroid** [comment]
@AllMightLegend Could you kindly explain why this PR is needed and how it reduces the size of the ghcr container?

In general it's a good idea to create an issue first explaining a problem, and then link your PR to that issue. Thanks!

**Comment by AllMightLegend** [comment]
I'm very new to open source contribution.So thanks for the advice.Actually if image is created through alpine the size will be less and less lines of code are always better

File: docker/dockerfile.pwa [hunk 1]
```diff
@@ -1,13 +1,6 @@
...
 
-COPY ./dist/manifest.json /usr/share/nginx/html
-COPY ./dist/replayWorker.js /usr/share/nginx/html
-COPY ./dist/service-worker.js /usr/share/nginx/html
-COPY ./dist/index.html /usr/share/nginx/html
-COPY ./dist/CHANGELOG.md /usr/share/nginx/html
-COPY ./dist/LICENSE /usr/share/nginx/html
-COPY ./dist/www /usr/share/nginx/html/www
-COPY ./dist/archives/README.md /usr/share/nginx/html/archives/README.md
+COPY ./dist .
```
Comment (Jaifroid) [inline docker/dockerfile.pwa:5]: Copying everything in dist isn't appropriate, as we may have files for different platforms there that are not relevant to the PWA in particular. It's better to be explicit about what you're copying.

**Review by Jaifroid — CHANGES_REQUESTED** [review]
Please see comments. Thanks!

File: .history/docker/dockerfile_20240810002439.pwa
[the remark this answers is not in the document]
Comment (AllMightLegend) [reply .history/docker/dockerfile_20240810002439.pwa:1]: Okay thanks

File: docker/dockerfile.pwa (hunk 1 above)
Comment (AllMightLegend) [reply docker/dockerfile.pwa:5]: Oh okay

**Pull request closed without merging**